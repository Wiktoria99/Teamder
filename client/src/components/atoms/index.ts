export * from './BackButton';
export * from './CustomTextField';
export * from './CustomButton';
export * from './SideBarButton';
