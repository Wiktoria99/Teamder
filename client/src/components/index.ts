export * from "./atoms";
export * from "./atoms";
export * from "./molecules";
export * from "./organisms";
