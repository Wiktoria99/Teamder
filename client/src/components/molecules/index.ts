export * from './RegisterSteps';
export * from './TeamItem';
export * from './Suggestions';
export * from './Sidebar';
export * from './TeamList';
export * from './SuggestionItem';
export * from './SuggestionList';
export * from './NotificationItem';
export * from './NotificationList';
