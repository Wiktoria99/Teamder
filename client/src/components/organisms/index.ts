export * from './Layout';
export * from './FormWrapper';
export * from './MainWrapper';
export * from './BarWrapper';
