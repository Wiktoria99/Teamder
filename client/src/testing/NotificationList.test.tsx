import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { NotificationItem, NotificationList } from '@/components';

it('NotificationItem renders without crashing', () => {
  const notif = {
    person: '',
    photoSource: '',
    title: '',
  };
  const div = document.createElement('div');
  ReactDOM.render(<NotificationItem notification={notif} />, div);
});

it('NotificationItem matches snapshot', () => {
  const notif = {
    person: '',
    photoSource: '',
    title: '',
  };
  const tree = renderer
    .create(<NotificationItem notification={notif} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('NotificationList renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<NotificationList />, div);
});

it('NotificationList matches snapshot', () => {
  const tree = renderer.create(<NotificationList />).toJSON();
  expect(tree).toMatchSnapshot();
});
