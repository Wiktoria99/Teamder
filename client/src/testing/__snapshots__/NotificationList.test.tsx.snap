// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationItem matches snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-15"
>
  <div
    className="MuiBox-root MuiBox-root-16 makeStyles-info-2"
  >
    <div
      className="MuiBox-root MuiBox-root-17 makeStyles-avatarContainer-3"
    >
      <img
        alt="avatar"
        className="makeStyles-photo-4"
        height="100"
        src=""
        width="100"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-18 makeStyles-nameContainer-5"
    >
      <p
        className="makeStyles-name-6"
      >
        <span>
          
        </span>
         chce dołączyć do twojego zespołu 
        <span>
          
        </span>
      </p>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-19 makeStyles-buttonContainer-7"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-8"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Akceptuj
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonDelete-9"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Odrzuć
      </span>
    </button>
  </div>
</div>
`;

exports[`NotificationList matches snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-32 makeStyles-notificationList-20"
>
  <div>
    <div
      className="MuiBox-root MuiBox-root-33"
    >
      <div
        className="MuiBox-root MuiBox-root-34 makeStyles-info-2"
      >
        <div
          className="MuiBox-root MuiBox-root-35 makeStyles-avatarContainer-3"
        >
          <img
            alt="avatar"
            className="makeStyles-photo-4"
            height="100"
            src="https://avatars.githubusercontent.com/u/65246171?s=400&v=4"
            width="100"
          />
        </div>
        <div
          className="MuiBox-root MuiBox-root-36 makeStyles-nameContainer-5"
        >
          <p
            className="makeStyles-name-6"
          >
            <span>
              Jan Kowalski
            </span>
             chce dołączyć do twojego zespołu 
            <span>
              Hackathon - Webdev and mobile applications! 
            </span>
          </p>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-37 makeStyles-buttonContainer-7"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-8"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Akceptuj
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonDelete-9"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Odrzuć
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
  <div>
    <div
      className="MuiBox-root MuiBox-root-38"
    >
      <div
        className="MuiBox-root MuiBox-root-39 makeStyles-info-2"
      >
        <div
          className="MuiBox-root MuiBox-root-40 makeStyles-avatarContainer-3"
        >
          <img
            alt="avatar"
            className="makeStyles-photo-4"
            height="100"
            src="https://avatars.githubusercontent.com/u/44270336?v=4"
            width="100"
          />
        </div>
        <div
          className="MuiBox-root MuiBox-root-41 makeStyles-nameContainer-5"
        >
          <p
            className="makeStyles-name-6"
          >
            <span>
              Irena Słowik
            </span>
             chce dołączyć do twojego zespołu 
            <span>
              Festiwal tańca i śpiewu
            </span>
          </p>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-42 makeStyles-buttonContainer-7"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-8"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Akceptuj
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonDelete-9"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Odrzuć
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
